### ISSUE TITLE
Comments from a squirrel

### ISSUE THREAD

**Issue by jhong93** [David-Durst/csknow#2]
L119: Change to AdamW or Adam (less need to tune LR)

L85: no relus after final layer

L95: check docs to see if you are applying softmax twice by accident from nn.CrossEntropy
 
    cross_entropy and cross_entropy_with_logits (check L119)
    https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

https://github.com/David-Durst/csknow/blob/f1551f49db10b4fec6a61c5d3623e24565fdc214/learn_bot/learn_bot/bot.py#L118

**Comment by David-Durst** [comment]
L119: changed. do I need to set any parameters? Or are the defaults all good?

L85: fixed

L95: I was, fixed

**Comment by David-Durst** [comment]
I notice that Adam gets the same test set avg loss after epoch 1 as SGD gets after epoch 5, but Adam never improves during epochs 2-5. I guess my loss is so small (0.04) that there's nothing more to learn?
